ops/services/app_functions/report_stream_batched_publisher/functions/package.json

```diff
@@ -7,8 +7,8 @@
     "coverage": "jest --coverage",
     "clean": "rimraf build",
     "build:compile": "tsc",
-    "build:localSetup": "yarn coverage && yarn prestart && npm prune --production && yarn build:zip",
-    "build:production": "yarn coverage && yarn build && npm prune --production && yarn build:zip",
+    "build:localSetup": "yarn && yarn coverage && yarn prestart && yarn install --production && yarn build:zip",
```

Comment:
shouldn't `yarn install` happen first thing, before running the `yarn coverage`, since it would need jest to be installed

Reply:
looks like the yarn === yarn install by default